mergekit/scripts/evolve.py

```diff
@@ -107,6 +107,12 @@
     default=True,
     help="Convert models to single-shard safetensors for faster merge",
 )
+@click.option(
+    "--timeout",
+    type=float,
+    default=None,
+    help="Maximum time to run the optimization in seconds",
```

Comment:
It might be worth documenting what happens when timeout is reached.  It should finalize the search and produce the best merged model at this point right?

Reply:
I believe that is what happens yep